images/buildkit/config/main.tf

```diff
@@ -28,6 +28,7 @@ output "config" {
     }
     environment = {
       BUILDKIT_SETUP_CGROUPV2_ROOT = 1
+      XDG_RUNTIME_DIR              = "/tmp"
```

Comment:
Can you add a comment explaining this?  Seems odd and it'd be good to record the context you have for "tomorrow Billy (or Matt or Jason)" :)

Reply:
Done.